main.go

```diff
@@ -520,46 +520,56 @@ func run(state overseer.State) {
 		return
 	}
 
-	topLevelSubCommand, _, _ := strings.Cut(cmd, " ")
-	switch topLevelSubCommand {
-	case analyzeCmd.FullCommand():
-		analyzer.Run(cmd)
-	default:
-		metrics, err := runSingleScan(ctx, cmd, engConf)
-		if err != nil {
-			logFatal(err, "error running scan")
-		}
-
-		verificationCacheMetrics := struct {
-			Hits                    int32
-			Misses                  int32
-			HitsWasted              int32
-			AttemptsSaved           int32
-			VerificationTimeSpentMS int64
-		}{
-			Hits:                    verificationCacheMetrics.ResultCacheHits.Load(),
-			Misses:                  verificationCacheMetrics.ResultCacheMisses.Load(),
-			HitsWasted:              verificationCacheMetrics.ResultCacheHitsWasted.Load(),
-			AttemptsSaved:           verificationCacheMetrics.CredentialVerificationsSaved.Load(),
-			VerificationTimeSpentMS: verificationCacheMetrics.FromDataVerifyTimeSpentMS.Load(),
-		}
-
-		// Print results.
-		logger.Info("finished scanning",
-			"chunks", metrics.ChunksScanned,
-			"bytes", metrics.BytesScanned,
-			"verified_secrets", metrics.VerifiedSecretsFound,
-			"unverified_secrets", metrics.UnverifiedSecretsFound,
-			"scan_duration", metrics.ScanDuration.String(),
-			"trufflehog_version", version.BuildVersion,
-			"verification_caching", verificationCacheMetrics,
-		)
-
-		if metrics.hasFoundResults && *fail {
-			logger.V(2).Info("exiting with code 183 because results were found")
-			os.Exit(183)
-		}
+	metrics, err := runSingleScan(ctx, cmd, engConf)
+	if err != nil {
+		logFatal(err, "error running scan")
+	}
+
+	verificationCacheMetricsSnapshot := struct {
+		Hits                    int32
+		Misses                  int32
+		HitsWasted              int32
+		AttemptsSaved           int32
+		VerificationTimeSpentMS int64
+	}{
+		Hits:                    verificationCacheMetrics.ResultCacheHits.Load(),
+		Misses:                  verificationCacheMetrics.ResultCacheMisses.Load(),
+		HitsWasted:              verificationCacheMetrics.ResultCacheHitsWasted.Load(),
+		AttemptsSaved:           verificationCacheMetrics.CredentialVerificationsSaved.Load(),
+		VerificationTimeSpentMS: verificationCacheMetrics.FromDataVerifyTimeSpentMS.Load(),
+	}
+
+	// Print results.
+	logger.Info("finished scanning",
+		"chunks", metrics.ChunksScanned,
+		"bytes", metrics.BytesScanned,
+		"verified_secrets", metrics.VerifiedSecretsFound,
+		"unverified_secrets", metrics.UnverifiedSecretsFound,
+		"scan_duration", metrics.ScanDuration.String(),
+		"trufflehog_version", version.BuildVersion,
+		"verification_caching", verificationCacheMetricsSnapshot,
+	)
+
+	if metrics.hasFoundResults && *fail {
+		logger.V(2).Info("exiting with code 183 because results were found")
+		os.Exit(183)
 	}
+
+	// Print results.
+	logger.Info("finished scanning",
```

Comment:
It looks like this is an unresolved merge conflict that prints the summary twice.

Reply:
Thanks for flagging this!

https://github.com/trufflesecurity/trufflehog/pull/3903